docs/development/releasing.rst

```diff
@@ -140,31 +161,30 @@ packages that use the full bugfix/maintenance branch approach.)
       $ pip install tox --upgrade
       $ TEST_READ_HUGE_FILE=1 tox -e test-alldeps -- --remote-data=any
 
-#. Render the changelog with towncrier, and confirm that the fragments can be
-   deleted. (Note: update this when doing the next release!)
-   ::
+#. We now need to render the changelog with towncrier. Since it is a good idea to
+   review the changelog and fix any line wrap and other issues, we do this on
+   a separate branch and open a pull request into the release branch to allow for
+   easy review. First, create and switch to a new branch based off the release
+   branch, e.g.::
 
-       towncrier [--draft] --version 4.3
+      $ git checkout -b v5.0-changelog
 
-#. Then add and commit those changes with::
+   Next, run towncrier and confirm that the fragments can be deleted::
 
-      $ git add CHANGES.rst
-      $ git commit -m "Finalizing changelog for v<version>"
+       towncrier --version 5.0
 
-#. Push the branch back to GitHub, e.g.::
+   Then add and commit those changes with::
 
-      $ git push upstream v1.2.x
-
-   and make sure that the CI services mentioned above (includnig the Azure pipeline)
-   are still passing.
-
-   .. note::
+      $ git add CHANGES.rst
+      $ git commit -m "Finalizing changelog for v<version>"
 
-      You may need to replace ``upstream`` here with ``astropy`` or
-      whatever remote name you use for the `astropy core repository`_.
+   Push to GitHub and open a pull request for merging this into the release branch,
+   e.g. v5.0.x.
 
```

Comment:
Unfortunately I don't think this works for anything other than an LTS release: any changes to the changeling need to be made in the *fragments* so that they are also seen in the LTS branch.

That said, this is fine for 5.0 so we can defer addressing it until 5.1...

Reply:
See my comment after your review about this.